terraform/modules/asm/scripts/install_mesh.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Verify variables
+echo -e "Project is ${PROJECT_ID}"
+echo -e "CLUSTER is ${CLUSTER}"
+echo -e "LOCATION is ${LOCATION}"
+echo -e "ASM_PACKAGE is ${ASM_PACKAGE}"
+
+# Setup kubeconfig
+export WORKDIR=`pwd`
+echo -e "Setting up kubeconfig at ${WORKDIR}/tempkubeconfig"
+export KUBECONFIG=${WORKDIR}/tempkubeconfig
+
+# Enable ASM Mesh which installs ASM CRDs
+echo -e "Enabling ASM Mesh on the GKE HUB"
+gcloud beta container hub mesh enable --project=${PROJECT_ID}
+
+NUM_MEMBERS=`gcloud beta container hub memberships list --project=${PROJECT_ID} --format="value(name)" | wc -l | awk '{print $1}'`
+echo -e "NUM_MEMBERS: ${NUM_MEMBERS}"
+STATE_CODE="OK"
+if [[ ${NUM_MEMBERS} -le 1 ]]; then
+    echo -e "STATE_CODE: ${STATE_CODE}"
+else
+    for i in {2..${NUM_MEMBERS}}; do
+        STATE_CODE="${STATE_CODE};OK"
+    done
+    echo -e "STATE_CODE: ${STATE_CODE}"
+fi
+
+for i in {1..900}; do
```

Comment:
hmm I wonder if there's a way to do a while-loop like this, without setting an arbitrary upper bound? 
also does it make sense to set a short `sleep` between iterations? 

maybe... 

```
while true
do
  <check status, if-else>
  sleep 5
done
```